src/training/predict.py

```diff
@@ -0,0 +1,85 @@
+import math
+import numpy as np
+import tifffile as tiff
+import os
+
+from train_unet import weights_path, get_model, normalize, PATCH_SZ, N_CLASSES
+
+if not os.path.exists('data/WV_predicted'):
+        os.makedirs('data/WV_predicted')
+
+image= normalize(tiff.imread('name of the WV image.tif').transpose([1,2,0]))

```

Comment:
I agree with @bspitzbart here. You pass this as a parameter. There are several way to do it. A quick one would be
```suggestion
import sys
image = normalize(tiff.imread(sys.argv[2]).transpose([1,2,0]))
```

When the user will do `python predict.py ImageName.tif` it will be passed immediately to `imread` and that will allow to run multiple images without the need to rewrite the script every time.